### DISCUSSION TITLE
post: GSAP 3 scrollTrigger Scrub and Pin

### DISCUSSION THREAD

**Discussion by giscus** [wghglory/my-website#66] [Announcements]
# post: GSAP 3 scrollTrigger Scrub and Pin

Guanghui Wang Website

https://derekw.vercel.app/posts/gsap-scroll-trigger-scrub-and-pin

**Comment by jimwilson94** [comment]
GSAP is amazing!

**Reply by wghglory** [reply]
yea great tool
